tsconfig.paths.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
```

Comment:
These are needed to wire up all the packages so TypeScript can do a build.

Reply:
Do we need the `..` one's? Removing them still results in functional stories + tests for me

Reply:
Cool, will remove.